src/routes/(console)/project-[project]/storage/bucket-[bucket]/settings/+page.ts

```diff
@@ -0,0 +1,10 @@
+import type { PageLoad } from './$types';
+import { sdk } from '$lib/stores/sdk';
+
+export const load: PageLoad = async ({ parent }) => {
+    const { organization } = await parent();
+    const currentPlan = await sdk.forConsole.billing.getPlan(organization.$id);
```

Comment:
We're probably going to be using the plan from `getPlan()` more and more across the console to replace how we've been working with plans before so it might be good to have this as a store. Maybe you can create an issue so we can revisit this later?